libs/sdk-ui-dashboard/src/presentation/widget/insight/DefaultDashboardInsight/InsightDrillDialog/InsightDrillDialog.tsx

```diff
@@ -45,6 +48,15 @@ export const InsightDrillDialog = (props: InsightDrillDialogProps): JSX.Element
 
     const executionsHandler = useWidgetExecutionsHandler(DRILL_MODAL_EXECUTION_PSEUDO_REF);
 
+    const { ErrorComponent, LoadingComponent } = useDashboardComponentsContext({
+        ErrorComponent: ThrowMissingComponentError("ErrorComponent"),
```

Comment:
Do we need to "reset" the ErrorComponent as well? Maybe we should just not mention the ErrorComponent here at all and just override the Loading one? If so, then ThrowMissingComponentError can go back to not being exported.

Reply:
Removed component overriding, it is not used in the child component and there is no need to do this.